### PR TITLE
adding anthropic models

### PR DESCRIPTION
**Pull request by himanshu-mishra3**
Following models are added under Anthropic ai service. 
1. Claude 3 Opus - claude-3-opus-20240229
2. Claude 3 Sonnet - claude-3-sonnet-20240229
3. Claude 3 Haiku - claude-3-haiku-20240307

### CODE REVIEW

**Review by trd-rmit — APPROVED** [review]

File: .env-sample [hunk 1]
```diff
@@ -95,5 +96,18 @@ AI_SERVICES_CONFIG='{
...
+            },
+            "claude-3-sonnet-20240229": {
+                "model_name": "your_anthropic_claude-3-sonnet-20240229_model_name_here"
+            },
+            "claude-3-haiku-20240307": {
+                "model_name": "your_anthropic_claude-3-haiku-20240307_model_name_here"
+            }
+        }
     }
 }'
```
Comment (trd-rmit) [inline .env-sample:113]: Please add a new line EOF.

File: library/aggrag/core/ai_service.py [hunk 2]
```diff
@@ -95,5 +104,7 @@ def get_ai_service(ai_service, llm_model=None, embed_model=None):
             return TogetherAIService(model=llm_model, embed_model=embed_model)
         elif ai_service == "OpenAI":
             return OpenAIService(model=llm_model, embed_model=embed_model)
+        elif ai_service == "Anthropic":
+            return AnthropicAIService(model=llm_model, embed_model=embed_model)
         else:
-            raise ValueError(f"Unsupported AI service: {ai_service}")
+            raise ValueError(f"Unsupported AI service: {ai_service}")
```
Comment (trd-rmit) [inline library/aggrag/core/ai_service.py:110]: Please add a new line EOF.

File: library/requirements.in [hunk 3]
```diff
@@ -20,4 +20,5 @@ llama-index-vector-stores-lancedb==0.1.7
 llama-index-packs-raptor
 llama-index-llms-together==0.1.3
 llama-index-llms-replicate==0.1.3
-replicate==0.31.0
\ No newline at end of file
+replicate==0.31.0
+llama-index-llms-anthropic==0.1.11
```
Comment (trd-rmit) [inline library/requirements.in:25]: Please add a new line EOF.

**Review by kangkan-fractal — APPROVED** [review]
LGTM

**Pull request merged**